### PR TITLE
Add snowflake snowsql 1.1.21

### PR DESCRIPTION
**Pull request by smtakeda**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/snowflake-snowsql.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
   version '1.1.21'
   sha256 '704d2b1b0f0e16ac0565fc14342dc5af7bc21e9ffa75575cef9e2c734dd0876d'
 
-  # s3-us-west-2.amazonaws.com was verified as official when first introduced to the cask
-  url "https://s3-us-west-2.amazonaws.com/sfc-snowsql-updates/bootstrap/1.1/darwin_x86_64/snowsql-#{version}-darwin_x86_64.pkg"
+  # s3-us-west-2.amazonaws.com/sfc-snowsql-updates/bootstrap was verified as official when first introduced to the cask
+  url "https://s3-us-west-2.amazonaws.com/sfc-snowsql-updates/bootstrap/#{version.major_minor}/darwin_x86_64/snowsql-#{version}-darwin_x86_64.pkg"
```
Comment (smtakeda) [inline Casks/snowflake-snowsql.rb]: Good to know this trick! Thanks for the patch.

**Review by smtakeda — COMMENTED** [review]
Looks good to me.

**Comment by smtakeda** [comment]
Would you accept this PR? Let me know if anything else I need to add or adjust. Thanks!

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**